tests/unit/src/components/TransactionList.spec.js

```diff
@@ -109,64 +119,45 @@ describe('Changes the current page when clicking next button in pagination', ()
 })
 
 describe('Renders the correct elements when click is not triggered', () => {
-  it('render the number of transactions', () => {
-    const wrapper = mount(TransactionList, getData())
+  test('render the number of transactions', () => {
+    const wrapper = mount(TransactionList, getData(87, 13))
 
     expect(wrapper.find('[data-test="transactions-length"]').text()).toContain(
-      'Transactions 87 transactions',
+      'Transactions87',
```

Comment:
should this contain a white space?

Reply:
No, becase it is a span that has a padding or a gap, it is not necessary